app/components/record_details/view.rb

```diff
@@ -106,7 +111,9 @@ def record_created_row
 
     def trainee_start_date_row
       start_date = RecordDetails::TraineeStartDate.new(trainee)
-
+      if start_date.course_empty? && trainee.created_from_hesa
```

Comment:
As above, I thought it was itt_end_date and not itt_start_date that was the optional one.

Reply:
cool! that's fine then, i was more worried that we weren't handling itt_end_date being missing, which we definitely need to (see above).

Reply:
Are we talking about ITT start date or commencement date? I think they're two separate items.